articles/openshift/concepts-networking.md

```diff
@@ -0,0 +1,114 @@
+
+OpenShift Software Defined Networking (SDN) is used to configure an overlay network using Open vSwitch (OVS), an OpenFlow implementation based on Container Network Interface (CNI) specification. The SDN supports different plugins, and the plugin being used in Azure Red Hat on OpenShift 4 is Network Policy. All network communication is managed by the SDN, so no extra routes are needed on your virtual networks to achieve pod to pod communication.
+
+### Azure Red Hat OpenShift Networking Specifics
+
+The following features are specific to Azure Red Hat OpenShift:
+* Bring your own virtual network is supported.
+* Pod and Service Network CIDRs are configurable.
+* Nodes and Masters are in different subnets.
+* Nodes and Masters virtual network subnets should be minimum /27.
+* Pod CIDR should be minimum /18 in size (The pod network is non-routable IPs, and is only used inside the OpenShift SDN).
+* Each node is allocated /23 subnet (512 IPs) for its pods (you can not change this value).
+* You can not attach a pod to multiple networks.
+* You can not configure Egress static IP. (This is an OpenShift feature. See the documentation on [configuring egress IPs](https://docs.openshift.com/aro/4/networking/openshift_sdn/assigning-egress-ips.html) for more details).
+
+### Network Modes
+
+The following network modes are available in Azure Red Hat OpenShift 4x:
+
+* **API Visibility** - Set the API visibility when running the [az aro create command](tutorial-create-cluster.md#create-the-cluster).
+    * "Public" - API Server is accessible by external networks.
+    * "Private" - API Server assigned a private IP from the masters subnet, only accessible using connected networks (peered virtual networks, other subnets in the cluster). A private DNS Zone will be created on the customer's behalf.
+* **Ingress Visibility** - Set the API visibility when running the [az aro create command](tutorial-create-cluster.md#create-the-cluster).
+    * "Public" routes will default to public Azure Standard Load Balancer (this can be changed).
+    * "Private" routes will default to internal load balancer (this can be changed).
+
+### Network Security Groups
+Network security groups will be created in the nodes Resource Group which is locked. The network security groups are assigned directly to the subnets not on the node's NICs. The network security groups are immutable, which means you do not have the permissions to change them. 
+
+Some things that are not supported:
+You can not create network security groups and assign them to the node's NICs.
+With a publicly visible API server, you can not create network security groups and assign them to the NICs.
+
+### DNS
+Azure Red Hat OpenShift uses CoreDNS. Domain forwarding can be configured (see the documentation on [using DNS forwarding](https://docs.openshift.com/aro/4/networking/dns-operator.html#nw-dns-forward_dns-operator) for more details.
+
+Currently you ca not bring your own DNS to your virtual networks.
```

Comment:
Currently, you can't bring your own DNS Resolvers (Custom DNS) to the virtual network (VNET) where Azure Red Hat Openshift Cluster was deployed, Azure Provided DNS should be used and is the default option. 

Note: the above should be highlighted as a warning.